app/services/true_layer/identify_applicant.rb

```diff
@@ -0,0 +1,53 @@
+module TrueLayer
+  class IdentifyApplicant
+    TRUE_LAYER_URL = 'https://api.truelayer.com'.freeze
+
+    attr_reader :token, :error
+
+    def initialize(token)
+      @token = token
+    end
+
+    def applicant
+      return login_error unless response.success?
+      Applicant.find_by(email: emails) || applicant_lookup_error
```

Comment:
🤔 I have a feeling that this might cause us more trouble right now than actually help us, as I'm not sure there's any  guarantee that the email address registered in our system is going to be the same as the one the applicant is currently using to login to TrueLayer/Online Bank Account.

**not necessarily a blocker**, but I would consider to review this identification process with Neil, to confirms what are the service needs.

Reply:
As discussed - I'll strip this out.